src/lib.rs

```diff
@@ -12,12 +12,12 @@
 #![feature(compiler_builtins)]
 #![feature(core_intrinsics)]
 #![feature(naked_functions)]
-#![feature(i128_type)]
 #![feature(repr_simd)]
 #![feature(abi_unadjusted)]
 #![feature(linkage)]
 #![feature(lang_items)]
 #![allow(unused_features)]
```

Comment:
Oh this is `#![allow(unused_features)]`, but you need `#[allow(stable_features)]`.